src/elements/hx-checkbox/_shadow.scss

```diff
@@ -10,7 +10,7 @@
   width: 100%;
 }
 
-/* icons */
```

Comment:
Removed raw CSS comment to reduce size of generated CSS output. This can be seen in several files for this PR.